UserInput.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using GameOfLife;
+
+namespace GameOfLife
+{
+    public class UserInput
+    {
+        /// <summary>
+        /// Returns an int of Height from user input.
+        /// </summary>
+        /// <returns></returns>
+        public static int UserInputHeight()
+        {
+            int x;
+            Console.WriteLine("Enter the height of the board.");
+            var temp = Console.ReadLine();
```

Comment:
1. Naming this variable temp is confusing could be something like userInput.
2. You don't need this temp variable at all you can get user input straight away in while condition like
```
while (!Int32.TryParse(Console.Readline(), out x))
````

Reply:
temp variable removed and UserInput method updated.